lib/_http_client.js

```diff
@@ -709,11 +709,7 @@ function responseOnEnd() {
 
   if (!req.shouldKeepAlive) {
     if (socket.writable) {
```

Comment:
this condition is not strictly necessary... remove for consistenct?

Reply:
Which one? `if (socket.writable)`?

Reply:
Yes